### DISCUSSION TITLE
blog/devlog-24/

### DISCUSSION THREAD

**Discussion by giscus** [modulocity/automationstationgame.com#9] [Comments]
# blog/devlog-24/

Hey folks! It’s been a long time coming, but I’ve finally added grass and bushes to the game. Long gone are the days of only looking at blubber trees on desolate islands. Also, arms are back! However, the majority of this sprint was focused on the interaction systems and UI changes to streamline gameplay. There is a lot of talk about so let’s get right into it.

https://automationstationgame.com/blog/devlog-24/

**Comment by ArjanBroos** [comment]
I'm a big fan of automation games and yours looks so charming! Looking forward to this, keep up the good work!

**Reply by ScottJDaley** [reply]
Thank you!!
